recipes/libaatm/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Could you please remove the shebang?
``conda-build`` already takes care of it! :)
```suggestion
```